.github/workflows/trigger-release-candidate-build-and-deploy.yml

```diff
@@ -46,3 +55,25 @@ jobs:
           token: ${{ secrets.GITHUB_TOKEN }}
           event-type: build-images
           client-payload: '{"ref": "refs/heads/staging"}'
+
+      - name: Send slack notification if main not merged into staging
+        if: failure()
+        uses: 8398a7/action-slack@v3
+        with:
+          status: ${{ job.status }}
+          fields: repo,commit,author,eventName,workflow,job,mention
+          mention: "here"
```

Comment:
just curious, how does the notification (textually) indicate that the merge-to-staging failed?

Reply:
These are the fields and their meaning: https://action-slack.netlify.app/usage/fields/

and this is what it looks like: https://action-slack.netlify.app/

<img width="1320" alt="Screenshot 2023-10-18 at 9 54 39 AM" src="https://github.com/chanzuckerberg/single-cell-data-portal/assets/5581819/2d67dab1-04f4-43c7-8a8a-a0b93c72e7e8">